### DISCUSSION TITLE
Render report from LHR for a specific metric

### DISCUSSION THREAD

**Discussion by thelovekesh** [GoogleChrome/lighthouse#14356] [Q&A]
Hello, 

I have been trying to render a report from LHR for a specific metric, let's say `performance`. Currently, when I use [`renderReport`](https://github.com/GoogleChrome/lighthouse/blob/master/report/renderer/report-renderer.js#L49) function it gives me HTML for all metric and it seems metric specif options are not there.

So, is there any workaround to output HTML for a specific metric only from LHR?

**Comment by connorjclark** [comment]
onlyCategories option is what you are looking for, to create an report with only certain categories. Or do you really need to render just a single category given a bigger LHR?

**Reply by thelovekesh** [reply]
> Or do you need to render just a single category given a bigger LHR

@connorjclark If there is a workaround then yes, this is the thing I am looking for.

**Reply by connorjclark** [reply]
In that case, you should (optionally clone the LHR and) delete the `.categories` you don't care about.
